java/packagescanner_test.go

```diff
@@ -10,6 +15,35 @@ import (
 	"github.com/quay/claircore/test"
 )
 
+// TestDisableAPI runs the java scanner over some layers known to have java
+// packages installed.
+func TestDisableAPI(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Error("http request called")
+	}))
+
+	ctx := context.Background()
+	ctx = zlog.Test(ctx, t)
+	scanner := &java.Scanner{}
+	var buf bytes.Buffer
+	cfg := &java.ScannerConfig{
+		DisableAPI:        true,
+		API:               "",
```

Comment:
Should this be `API: srv.URL`? Otherwise we never trigger the expected test failure (we just trigger the results parity failure).

Reply:
🤦 Thank you